invadedhouse/commands.py

```diff
@@ -145,5 +152,31 @@ def interpret_command(self, cmd: str):
                     print(f'{item_str} is empty - nothing to loot!')
             else:
                 print('Can\'t loot. Try to loot a different loot.')
+        elif cmd == 'smash a wall' or cmd == 'smash the wall':
+            place_that_the_hole_leads_to = self.house.find_room_by_name('outside')
+            room_that_lets_you_into_the_sword_room = self.house.find_room_by_name('entry room')
+            room_with_a_wall_that_can_be_smashed = self.house.find_room_by_name('sword room')
+            if not self.player.get_location() == room_with_a_wall_that_can_be_smashed:
+                print('Smash attempt failed.')
+                return
+            else:
+                print('The wall has been smashed! You walk through your hole in the wall and you are outside.')
+                print('')
+                room_with_a_wall_that_can_be_smashed.set_exits([room_that_lets_you_into_the_sword_room, place_that_the_hole_leads_to])
+        elif cmd in('stab me', 'stab myself'):
+            cprint('Do you really want to stab yourself?', 'red')
+            are_you_sure = input('> ')
+            if are_you_sure in('yes', 'y'):
+                cprint('Okay...', 'red')
+                self.player.take_damage(self.player.get_hp())
```

Comment:
my super cool new function in action

Reply:
`self.player.take_damage(self.player.get_hp())`

Reply:
in here it makes the player take as much damage as their hp so they will just die

Reply:
we probably shouldn't let the player commit suicide, but maybe there is another way to accidentally take damage if you do something dangerous... like if you try to eat a weapon or something. we need to keep our game PG rated! 😇

Reply:
oh ok got it

Reply:
okay i fixed the command it doesnt sound evil anymore

Reply:
fixed

Reply:
FIXED IT!

Reply:
umm excuse me?

Reply:
LOOK I FIXED IT!!!!!!!!!!!!!!!!!!!!!!!!!!!! LOOOOOOOK

Reply:
look. at. it. please. because. i. fixed. it.

Reply:
LOOOOOOOOOOOOOOOOOOOK AT IT!

Reply:
look at it